### PR TITLE
Ariel Choquetopa - Add song lyrucs

### PR DESCRIPTION
**Pull request by ArielPuma747**
Song: warning sign
Artist: coldplay

### CODE REVIEW

**Comment by ArielPuma747** [comment]
@noemiguzo can you review this PR?

File: ariel-choquetopa.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+warning sign - COLDPLAY
+
+A warning signal
```
Comment (noemiguzo) [inline ariel-choquetopa.txt]: signal es sign

File: ariel-choquetopa.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+I missed the good part, then You realized
+I started looking and the bubble burst
+I started looking for excuses
+Come on in
+I've gotta tell yeah what a state I'm in
```
Comment (noemiguzo) [inline ariel-choquetopa.txt]: yeah es you

File: ariel-choquetopa.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+I've gotta tell you in my loudest tones
+That I started looking for a warning signed
```
Comment (noemiguzo) [inline ariel-choquetopa.txt]: signed es sign

File: ariel-choquetopa.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+When the truth is
+I miss you
+Yeah the truth is
```
Comment (noemiguzo) [inline ariel-choquetopa.txt]: falta ',' despues de Yeah

File: ariel-choquetopa.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
+warning sign - COLDPLAY
+
+A warning signal
+I missed the good part, then You realized
```
Comment (noemiguzo) [inline ariel-choquetopa.txt]: quitar ','
You realized es 'I realised'

**Review by noemiguzo — CHANGES_REQUESTED** [review]

File: ariel-choquetopa.txt (hunk 1 above)
Comment (ArielPuma747) [reply ariel-choquetopa.txt]: Done

File: ariel-choquetopa.txt (hunk 2 above)
Comment (ArielPuma747) [reply ariel-choquetopa.txt]: Done

File: ariel-choquetopa.txt (hunk 3 above)
Comment (ArielPuma747) [reply ariel-choquetopa.txt]: Done

File: ariel-choquetopa.txt (hunk 4 above)
Comment (ArielPuma747) [reply ariel-choquetopa.txt]: Done

File: ariel-choquetopa.txt (hunk 5 above)
Comment (ArielPuma747) [reply ariel-choquetopa.txt]: Done

**Comment by ArielPuma747** [comment]
@noemiguzo can you review this PR? is updated with your observation

**Review by noemiguzo — APPROVED** [review]

**Comment by noemiguzo** [comment]
Approved

**Pull request merged**